hyperjumptech/monika
# Add Documentation to --force

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Update missing docs to include `--force` and impolement 

## How did you implement / how did you fix it  
1.  add flag.force detection to `--flush` handler
2. add force documentation on cli-options.md

## How to test  
1. type: `bin/run --flush` should get Y/n confirmation
2. type: `bin --flush --force` should be flushed without confirmation
